addons/account/models/account.py

```diff
@@ -948,7 +948,7 @@ def _get_bank_statements_available_sources(self):
 
     # alias configuration for journals
     alias_id = fields.Many2one('mail.alias', string='Alias', copy=False)
-    alias_domain = fields.Char('Alias domain', compute='_compute_alias_domain', default=lambda self: self.env["ir.config_parameter"].sudo().get_param("mail.catchall.domain"))
+    alias_domain = fields.Char('Alias domain', compute='_compute_alias_domain', default=lambda self: self.env["ir.config_parameter"].sudo().get_param("mail.catchall.domain"), compute_sudo=True)
```

Comment:
I think we could remove the default (as computed fields should not have default), and normally compute sudo should not force you to re-use sudo in the compute itself.

Reply:
The default is quite useful here, when you create a new record from the web client, the compute won't be call and then the alias_domain will be empty during the creation (and it is case particularly, we can't access to this setting if there is no default). No ?
Indeed, the sudo is duplicate here.

Reply:
It is probably because there is no trigger ... isn't it possible to add a trigger, even if it is a fake one, so that we remove default ?